components/movementsensor/rtkutils/ntrip.go

```
@@ -60,3 +92,113 @@ func NewNtripInfo(cfg *NtripConfig, logger logging.Logger) (*NtripInfo, error) {
 	logger.Debug("Returning n")
 	return n, nil
 }
+
+// ParseSourcetable gets the sourcetable and parses it.
+func (n *NtripInfo) ParseSourcetable() (*Sourcetable, error) {
+	reader, err := n.Client.GetSourcetable()
+	if err != nil {
+		return nil, err
+	}
+	defer func() {
+		if err := reader.Close(); err != nil {
+			log.Println("Error closing reader:", err)
+		}
+	}()
+
+	st := &Sourcetable{}
+	st.Streams = make([]Stream, 0, 200)
+	scanner := bufio.NewScanner(reader)
+	ln := ""
+
+Loop:
+	for scanner.Scan() {
+		ln := scanner.Text()
+		if strings.HasPrefix(ln, "#") || strings.HasPrefix(ln, "*") { // comment
+			continue
+		}
+		fields := strings.Split(ln, ";")
+		switch fields[0] {
+		case "CAS":
+			continue
+		case "NET":
+			continue
+		case "STR":
+			if str, err := ParseStream(ln); err == nil {
```

Why ignore the error?